packages/core/rpc/BaseRpcDriver.ts

```diff
@@ -187,10 +187,32 @@ export default abstract class BaseRpcDriver {
       pluginManager,
       sessionId,
     )
-    const result = await worker.call(functionName, filteredAndSerializedArgs, {
-      timeout: 5 * 60 * 1000, // 5 minutes
-      ...options,
-    })
+
+    const channel = `message-${shortid.generate()}`
+    const listener = (message: string) => {
+      if (args.statusCallback) {
+        args.statusCallback(message)
+      }
+    }
+    // @ts-ignore
+    if (worker.on) {
+      // @ts-ignore
+      worker.on(channel, listener)
+    }
+
+    const result = await worker.call(
+      functionName,
+      { ...filteredAndSerializedArgs, channel },
+      {
+        timeout: 5 * 60 * 1000, // 5 minutes
+        ...options,
+      },
+    )
+    // @ts-ignore
+    if (worker.off) {
+      // @ts-ignore
```

Comment:
any way we can clean up these ignores?

Reply:
I moved the logic for the status callback into the webworker driver itself to try to handle this issue